stable/admin/values.yaml

```diff
@@ -112,6 +112,9 @@ admin:
     policies:
       nearest: random(first(label(pod ${pod:-}) label(node ${node:-}) label(zone ${zone:-}) any))
   
+  # Label for insights configuration.
+  insights: "insights-ap-plugin" 
```

Comment:
It looks like that this is a specific name of an Insights plugin. Can we document which plugins are available and the names?

Reply:
Thanks. This is even better. I recommend we document the possible plugins that we have and mention that other custom plugins can be created. Maybe also change the `insights` key to something like `insights-plugin` which would be more descriptive I think.